blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/TokenDiscoveryIO.java

```diff
@@ -0,0 +1,12 @@
+package com.rackspacecloud.blueflood.io;
+
+
+import com.rackspacecloud.blueflood.types.Token;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface TokenDiscoveryIO  extends BaseDiscoveryIO {
```

Comment:
The only reason `BaseDiscoveryIO` is present, it because I have `getMetricNames` that is being supported in both `DiscoverIO` and `TokenDiscoveryIO`, one using aggregated ES queries and the other using separate ES index respectively. 

I was able to leverage this common interface while writing tests. I only write one test and use both implementations so that they both work the same.